Parsers/ASimAuthentication/Parsers/imAuthentication.yaml

```diff
@@ -50,8 +80,9 @@ ParserQuery: |
       , vimAuthenticationBarracudaWAF                   (starttime, endtime, targetusername_has, (imAuthenticationDisabled or('ExcludevimAuthenticationBarracudaWAF' in (DisabledParsers) )))
       , vimAuthenticationVectraXDRAudit               (starttime, endtime, (imAuthenticationDisabled or('ExcludevimAuthenticationVectraXDRAudit' in (DisabledParsers) )))
       , vimAuthenticationSentinelOne                    (starttime, endtime, (imAuthenticationDisabled or('ExcludevimAuthenticationSentinelOne' in (DisabledParsers) )))
+      , vimAuthenticationVMwareCarbonBlackCloud         (starttime=starttime, endtime=endtime, eventtype_in=eventtype_in, eventresultdetails_in=eventresultdetails_in, eventresult=eventresult, targetusername_has_any=targetusername_has_any, targetappname_has_any=targetappname_has_any, actorusername_has_any=actorusername_has_any, srcipaddr_has_any_prefix=srcipaddr_has_any_prefix, srchostname_has_any=srchostname_has_any, targetipaddr_has_any_prefix=targetipaddr_has_any_prefix, dvcipaddr_has_any_prefix=dvcipaddr_has_any_prefix, dvchostname_has_any=dvchostname_has_any, disabled=(imAuthenticationDisabled or('ExcludevimAuthenticationVMwareCarbonBlackCloud' in (DisabledParsers) )))
```

Comment:
@jayeshprajapaticrest @niralishah-crest regatrding validation error. Can you please update this for existing parsers entires?

starttime=starttime, endtime=endtime, targetusername_has=targetusername_has_any, disabled=

Reply:
Done